src/components/FeatureViews/Availability/CreateOOOForm.js

```diff
@@ -194,38 +195,62 @@ export default function CreateOOOForm() {
-                                    width: '150px',
-                                }}
-                                onClick={() =>
-                                    SubmitNoticeChild.current.handleClearForm()
-                                }
-                                color="primary"
-                                variant="outlined"
-                                data-cy="clear-OOO-button"
-                            >
-                                Clear
-                            </Button>
+                            <ResponsiveButton
+                            // style={{
+                            //     backgroundColor: 'white',
+                            //     color: 'black',
+                            //     width: '150px',
+                            // }}
+                            onClick={() =>
+                                SubmitNoticeChild.current.handleClearForm()
+                            }
+                            // color="primary"
+                            variant="outlined"
+                            data-cy="clear-OOO-button"
+                        >
+                            clear
+                        </ResponsiveButton>
+                        
+                            // <Button
+                            //     style={{
+                            //         backgroundColor: 'white',
+                            //         color: 'black',
+                            //         width: '150px',
+                            //     }}
+                            //     onClick={() =>
+                            //         SubmitNoticeChild.current.handleClearForm()
+                            //     }
+                            //     color="primary"
+                            //     variant="outlined"
+                            //     data-cy="clear-OOO-button"
+                            // >
+                            //     Clear
+                            // </Button>
                         )}
                     </Grid>
                     <Grid item>
                         {!submitted && (
-                            <Button
-                                style={{
-                                    backgroundColor: omouBlue,
-                                    color: 'white',
-                                    width: '150px',
-                                }}
+                            <ResponsiveButton
                                 onClick={handleSubmit}
-                                color="primary"
+                                // color="primary"
                                 variant="outlined"
                                 data-cy="submit-OOO-button"
                             >
-                                Submit
-                            </Button>
+                                submit
+                            </ResponsiveButton>
+                            // <Button
```

Comment:
No reason to save the old button,
"Let the past die! Kill it if you have to!"

Reply:
TIL I am Rey